pkg/serializer/internal/metrics/origin_mapping.go

```diff
@@ -320,6 +324,8 @@ func metricSourceToOriginCategory(ms metrics.MetricSource) int32 {
 		metrics.MetricSourceQuarkus,
 		metrics.MetricSourceMilvus:
 		return 11 // integrationMetrics
+	case metrics.MetricSourceGPU:
+		return 72 // ref: https://github.com/DataDog/dd-source/blob/main/domains/metrics/shared/libs/proto/origin/origin.proto#L424
```

Comment:
```suggestion
		return 72 // ref: https://github.com/DataDog/dd-source/blob/276882b71d84785ec89c31973046ab66d5a01807/domains/metrics/shared/libs/proto/origin/origin.proto#L427
```
Should be a permalink IMO